documentation/localization_howto.md

```diff
@@ -0,0 +1,165 @@
+    tor_locale: "fr"                # edge case for Tor          
+```
+
+Localizing gateway instructions
+-------------------------------
+
+The following roles contain user instructions that will require translation:
+
+ - `playbooks/roles/l2tp-ipsec/templates/`
+  1. `instructions.md.j2`
+
+ - `playbooks/roles/openconnect/templates`
+  1. `instructions.md.j2`
+  1. `mirror.md.j2`
+
+ - `playbooks/roles/openvpn/templates`
+  1. `instructions.md.j2`
+  1. `stunnel-insructions.md.j2`
+  1. `mirror.md.j2`
+
+ - `playbooks/roles/shadowsocks/templates`
+  1. `instructions.md.j2`
+  1. `mirror.md.j2`
+
+- `playbooks/roles/ssh-forward/templates`
+  1. `instructions.md.j2`
+  1. `mirror.md.j2`
+
+ - `playbooks/roles/streisand-gateway/templates`
+  1. `index.md.j2`
+  1. `instructions.md.j2`
+  1. `firewall.md.j2`
+
+ - `playbooks/roles/streisand-mirror/templates`
+  1. `mirror-index.md.j2`
+
+ - `playbooks/roles/stunnel/templates`
+  1. `mirror.md.j2`
+
+ - `playbooks/roles/tor/templates`
+  1. `instructions.md.j2`
+  1. `mirror.md.j2`
+
+ - `playbooks/roles/wireguard/templates`
+  1. `instructions.md.j2`
+
+Language selection in generated pages
+-------------------------------------
+
+By default, Streisand's common header files contains language selection 
+(generated using the `streisand_languages` dict), and for most pages will be done
+automatically. Certain edge cases do exist where a jinja2 code block is
+explicitly needed with the generated html filename.
+
+These files all reside with the `streisand-gateway` role:
+
+ - `firewall.md.j2`
+ - `index.md.j2`
+ - `instructions.md.j2`
+
+```
+- - -
+{% for key, value in streisand_languages.items() %}
+  [{{ value.language_name }}]({{ streisand_server_name }}-*generated-html-file-name*{{ value.file_suffix }}.html)&nbsp;
+{% endfor %}
+- - -
+```
+
+The HTML filename can be recovered by looking at the relevant ansible task that generates the
+template.
+
+End user considerations
+-----------------------
+
+While we aim to have as-close-to-the-original-source translations, linguistic differences in
+expressions should be taken into consideration when it makes.
+
+A non-technical example would be a the following
+expression in English: "the straw that broke the camel's back".
+A literal translation into French
+would give: "la paille qui a cassé le dos du chameau", however contextually
+a similar expression exists in French: "C'est la goutte d'eau qui fait déborder le vase"
+(it's the drop of water that made the vase overflow). Different translations,
+same meaning. In this case, the latter be the 'ideal' translation, as it would make more sense
+to a native speaker of that language.
+
+Another aspect to consider is in what language an end-user's device is set to.
+It is best to have a device nearby set in the said language, where you can see exactly what is
+displayed in particular menus and use those same exact strings.
```

Comment:
I think you can drop the "exact" in "exact strings" and its a little more natural reading since you already said "exactly what is displayed"